as/cNFT/assembly/models/persistent_tokens.ts

```diff
@@ -128,6 +172,18 @@ export class PersistentTokens {
 
         return accountTokenSet
     }
+
+    private _deleteFromAccountTokenSet(
+        tokenId: TokenId,
+        accountId: AccountId
+    ): void {
+        let accountTokenSet = this._amap.getSome(accountId)
+
+        if(accountTokenSet) {
+            accountTokenSet.delete(tokenId);
+            this._amap.set(accountId, accountTokenSet);
+        }
+    }
```

Comment:
Shouldn't this return `accountTokenSet`?

Reply:
I have already set the new tokens using line no 184. So no need to return them I suppose.

Reply:
Ah okay that makes sense, thanks!